tests/ReflectionHelpers.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Illuminate\Tests;
+
+use ReflectionClass;
+use ReflectionException;
+use ReflectionObject;
+
+trait ReflectionHelpers
+{
+    /**
+     * @param $obj
+     * @param $name
+     * @param array $args
+     * @return mixed
+     * @throws ReflectionException
+     */
+    protected static function privateCall($obj, $name, array $args = [])
+    {
+        $class = new ReflectionClass($obj);
+        $method = $class->getMethod($name);
+        $method->setAccessible(true);
+
+        return $method->invokeArgs($obj, $args);
+    }
+
+    /**
+     * @param $obj
+     * @param $name
+     * @return mixed
+     * @throws ReflectionException
+     */
+    protected static function getPrivateProperty($obj, $name)
+    {
+        $class = new ReflectionObject($obj);
+        $property = $class->getProperty($name);
+        $property->setAccessible(true);
+
+        return $property->getValue($obj);
+    }
+
+    /**
+     * @param mixed $object
+     * @param mixed $property
+     * @param $value
+     * @throws ReflectionException
+     */
+    protected function setPrivateProperty($object, $property, $value)
+    {
+        $reflection = new ReflectionClass($object);
+        $reflectionProperty = $reflection->getProperty($property);
+        $reflectionProperty->setAccessible(true);
+        $reflectionProperty->setValue($object, $value);
+    }
+}
```

Comment:
This class probably best belongs in a new PR alongside the new tests. Please remove this from this PR.

Also: a class like this probably belongs in the `Illuminate\Testing` component.

Reply:
The docblocks here also don't match the rest of the framework's docblocks style and formatting.